packages/expo-modules-autolinking/package.json

```diff
@@ -42,6 +42,7 @@
     "tempy": "^0.7.1"
   },
   "dependencies": {
+    "@expo/config": "~8.0.0",
```

Comment:
I hope this won't cause dependency versions issues

Reply:
we also have this in `@expo/cli`. given the current state where expo package has exact version to expo-modules-core, @expo/cli, @expo/config, and expo-modules-autolinking. i think it would be safe.